asciidocapi.py

```diff
@@ -189,16 +192,23 @@ def __init__(self, asciidoc_py=None):
             if not os.path.isfile(cmd):
                 raise AsciiDocError('missing file: %s' % cmd)
         else:
-            # Try shell search paths.
+            # try to find sibling
+            THIS_PATH = os.path.dirname(os.path.realpath(__file__))
```

Comment:
The normal Asciidoc convention is for variables to be lower case `this_path`

Reply:
since this variable is const during all runtime, i was planing to make it const (thus upper case in convention) but probably too much trouble : `this_path`  is good.